src/background/hypothesis-chrome-extension.js

```diff
@@ -120,14 +120,28 @@ export default function HypothesisChromeExtension({
     });
   }
 
+  /**
+   *
+   * @param {number} tabId
+   * @param {import('./tab-state').State|undefined} current
+   */
   function onTabStateChange(tabId, current) {
     if (current) {
-      browserAction.update(tabId, current);
-      chromeTabs.get(tabId, updateTabDocument);
+      chromeTabs.get(tabId, tab => {
+        // This error is raised if the tab doesn't exists.
```

Comment:
```suggestion
        // This error is raised if the tab doesn't exist.
```

Are there any other reasons this error might be raised that you know of?